### PR TITLE
chore: bump cloud-node-manager image v1.25.18, v1.26.14, v1.27.8 and …

### PR DESCRIPTION
**Pull request by nilo19**
…v1.28.0

**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

chore: bump cloud-node-manager image v1.25.18, v1.26.14, v1.27.8 and v1.28.0

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Comment by nilo19** [comment]
##[error]Artifact scenario-logs already exists for build 78613173.
This error is not related to the change.

File: vhdbuilder/packer/generate-windows-vhd-configuration.ps1 [hunk 1]
```diff
@@ -69,17 +69,16 @@ $global:imagesToPull += @(
     "mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver:v1.3.4",
     "mcr.microsoft.com/oss/azure/secrets-store/provider-azure:v1.4.1",
     # Azure cloud node manager. Owner: nilo19 (Qi Ni)
-    "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.24.18", # for k8s 1.24.x
-    "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.24.20", # for k8s 1.24.x
-    "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.24.21", # for k8s 1.24.x
-    "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.25.12", # for k8s 1.25.x
     "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.25.14", # for k8s 1.25.x
```
Comment (AbelHu) [inline vhdbuilder/packer/generate-windows-vhd-configuration.ps1:72]: could we remove v1.25.14?

Comment (AbelHu) [reply vhdbuilder/packer/generate-windows-vhd-configuration.ps1:72]: v1.26.10 and v1.27.4

Comment (nilo19) [reply vhdbuilder/packer/generate-windows-vhd-configuration.ps1]: removed

**Review by AbelHu — APPROVED** [review]
LGTM. Thanks for the PR

**Pull request merged**